src/RoleObjects/ButtonData.js

```diff
@@ -25,67 +27,68 @@ export default class ButtonData extends AccessibilityObject {
    * @inheritdoc
    */
   addChildAt(displayObject, index) {
-    if (!displayObject.accessible
+    if (
+      !displayObject.accessible
       || doesRoleUseSemanticallyInteractiveTag(displayObject.accessible.role)
-      || !_.isUndefined(displayObject.accessible.tabIndex)) {
+      || !_.isUndefined(displayObject.accessible.tabIndex)
+    ) {
       throw new Error(`Children of ${this.role} cannot be interactive`);
     }
     super.addChildAt(displayObject, index);
   }
 
   /**
-  * Sets whether the element should get pressed on page load
-  * @access public
-  * @param {boolean} val - true if button pressed
-  */
+   * Sets whether the element should get pressed on page load
+   * @access public
+   * @param {boolean} val - true if button pressed
+   */
   set pressed(val) {
     this._reactProps['aria-pressed'] = val;
   }
 
   /**
-  * Retrieves whether button pressed
-  * @access public
-  * @returns {boolean} true if button pressed
-  */
+   * Retrieves whether button pressed
+   * @access public
+   * @returns {boolean} true if button pressed
+   */
   get pressed() {
     return this._reactProps['aria-pressed'];
   }
 
-
   /**
-  * Sets whether the element should get expanded on page load
-  * @access public
-  * @param {boolean} val - true if button expanded, false if button not expanded,
-  *  undefined if the field is unset
-  */
+   * Sets whether the element should get expanded on page load
```

Comment:
```suggestion
   * Sets whether the element should get expanded
```
some tech debt pointed out by @mnewcomb314 (`on page load` shouldn't be there)

Reply:
I think we can capture it as next task/tech debt, since this is not scoped for the current task

Reply:
Changed as per the suggestion